src/main/java/uk/gov/justice/hmpps/probationteams/controllers/PingEndpoint.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.hmpps.probationteams.controllers;
+
+import org.springframework.boot.actuate.endpoint.annotation.ReadOperation;
+import org.springframework.boot.actuate.endpoint.web.annotation.WebEndpoint;
+import org.springframework.context.annotation.Configuration;
+
+import static org.springframework.http.MediaType.TEXT_PLAIN_VALUE;
+
+@Configuration
+@WebEndpoint(id = "ping")
+public class PingEndpoint {
+
+    @ReadOperation(produces = TEXT_PLAIN_VALUE)
+    public String ping() {
+        return "pong";
```

Comment:
ideally don't need this class - under spring boot 2.2 dependent apps should call `/health/ping` instead.  We've got a ticket to move it all but haven't progressed it yet, but would be great if all new apps therefore didn't have a `/ping`...